src/handleApps.ts

```diff
@@ -47,11 +47,12 @@ export async function handleApps(
 
   const finalResponse = new Response(proxyResponse.body, proxyResponse);
   const additionalHeaders = {
+    "cache-control": orgSettings.staticFiles.cacheControl,
     ...corsHeaders(request, orgSettings),
     ...foundryVersion(),
   };
   for (let additionalHeader in additionalHeaders) {
-    finalResponse.headers.append(
+    finalResponse.headers.set(
```

Comment:
I changed `append` to `set` to override the default cache-control value.
Let me know if there's a better way to go about this.

with append, it looked like this:
![image](https://user-images.githubusercontent.com/72050425/161343330-0c3fe5cf-1e72-4d73-9ac6-dafd45de38cb.png)

with set, it looks like this:
![image](https://user-images.githubusercontent.com/72050425/161343392-f833c77e-6358-452d-b56b-e7e932f17c3d.png)